.github/workflows/CI.yml

```diff
@@ -7,15 +12,15 @@ on:
 
 env:
   __TAUCMDR_PROGRESS_BARS__: "disabled"
-  __TAUCMDR_SYSTEM_PREFIX__: "~/taucmdr/system"
+  __TAUCMDR_SYSTEM_PREFIX__: "system"
```

Comment:
There's no good/easy way to de-reference `$HOME` or equivalent, so to begin with the variables are set and understood to be relative paths. Further down, we make them absolute and set them in each job, and also make them available in the context of expressions (i.e., `${{ ... }}`) so that they can be passed as arguments to actions.